mustermann/spec/mapper_spec.rb

```diff
@@ -19,21 +19,14 @@
     end
 
     context 'accepts mappings followed by options' do
-      subject(:mapper) { Mustermann::Mapper.new("/foo" => "/bar", additional_values: :raise) }
-      its(:to_h) { should be == { Mustermann.new("/foo") => Mustermann::Expander.new("/bar") } }
-      example { mapper['/foo'].should be == '/bar' }
-      example { mapper['/fox'].should be == '/fox' }
-    end
-
-    context 'accepts options followed by mappings' do
-      subject(:mapper) { Mustermann::Mapper.new(additional_values: :raise, "/foo" => "/bar") }
+      subject(:mapper) { Mustermann::Mapper.new({ "/foo" => "/bar" }, additional_values: :raise) }
       its(:to_h) { should be == { Mustermann.new("/foo") => Mustermann::Expander.new("/bar") } }
       example { mapper['/foo'].should be == '/bar' }
       example { mapper['/fox'].should be == '/fox' }
     end
 
     context 'allows specifying type' do
-      subject(:mapper) { Mustermann::Mapper.new(additional_values: :raise, type: :rails, "/foo" => "/bar") }
+      subject(:mapper) { Mustermann::Mapper.new({ "/foo" => "/bar" }, additional_values: :raise, type: :rails) }
```

Comment:
Do we need to update an example [here](https://github.com/sinatra/mustermann/blob/d103fbdfafc4b2148c76097301341b6e006cc3e6/mustermann/lib/mustermann/mapper.rb#L43)?